### PR TITLE
Add a `/check-required-labels` command

### PR DESCRIPTION
**Pull request by Halamix2**
Hopefully this will help with #22208

I have no idea how tests could look like for the comment handling though

### CODE REVIEW

File: prow/plugins/require-matching-label/require-matching-label.go [hunk 1]
```diff
@@ -244,3 +256,49 @@ func handle(log *logrus.Entry, ghc githubClient, cp commentPruner, configs []plu
...
+	repo := e.Repo.Name
+	number := e.Number
+
+	event := &event{
+		org:    org,
+		repo:   repo,
+		number: number,
+		author: e.User.Login,
+	}
+	if e.IsPR {
```
Comment (stevekuznetsov) [inline prow/plugins/require-matching-label/require-matching-label.go:289]: The `handle()` logic already knows how to pull labels down - why do we need this here?

Comment (Halamix2) [reply prow/plugins/require-matching-label/require-matching-label.go:289]: My bad, I've simplified the function to omit adding currentLabels field

**Comment by stevekuznetsov** [comment]
@Halamix2 cool! The code looks good, but you'll need to sign the CLA for us to accept the change (and run CI on it!). Please ping me when you've done so - we can merge this then.

**Comment by Halamix2** [comment]
I've signed it

**Comment by Halamix2** [comment]
I signed it

**Comment by Halamix2** [comment]
@stevekuznetsov how can I retrigger CLA check after signing?

**Comment by Halamix2** [comment]
I signed it

**Comment by Halamix2** [comment]
I signed it

**Comment by Halamix2** [comment]
Ah, there are two different CLA systems in place, that's what confused me

**Comment by fejta** [comment]
Please pass unit tests
/retest

File: prow/plugins/require-matching-label/require-matching-label.go [hunk 2]
```diff
@@ -95,14 +100,21 @@ func helpProvider(config *plugins.Configuration, _ []config.OrgRepo) (*pluginhel
...
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: `The require-matching-label plugin is a configurable plugin that applies a label to issues and/or PRs that do not have any labels matching a regular expression. An example of this is applying a 'needs-sig' label to all issues that do not have a 'sig/*' label. This plugin can have multiple configurations to provide this kind of behavior for multiple different label sets. The configuration allows issue type, PR branch, and an optional explanation comment to be specified.`,
+		Config: map[string]string{
+			"": fmt.Sprintf("The plugin has the following configurations:\n<ul><li>%s</li></ul>", strings.Join(descs, "</li><li>")),
 		},
-		nil
+		Snippet: yamlSnippet,
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/check-require-labels",
```
Comment (stevekuznetsov) [inline prow/plugins/require-matching-label/require-matching-label.go:111]:
```suggestion
		Usage:       "/check-required-labels",
```

nit: this reads as more correct English

File: prow/plugins/require-matching-label/require-matching-label.go [hunk 3]
```diff
@@ -95,14 +100,21 @@ func helpProvider(config *plugins.Configuration, _ []config.OrgRepo) (*pluginhel
...
+		Description: "Forces rechecking of the required labels.",
```
Comment (stevekuznetsov) [inline prow/plugins/require-matching-label/require-matching-label.go:112]:
```suggestion
		Description: "Checks for required labels.",
```

I'm not sure that "forces" means much for users who are not familiar with the internals of how this plugin usually work

**Review by stevekuznetsov — COMMENTED** [review]
Couple small nits!

File: prow/plugins/require-matching-label/require-matching-label.go (hunk 3 above)
Comment (Halamix2) [reply prow/plugins/require-matching-label/require-matching-label.go]: Done

File: prow/plugins/require-matching-label/require-matching-label.go (hunk 2 above)
Comment (Halamix2) [reply prow/plugins/require-matching-label/require-matching-label.go]: `check-require-labels` changed to `check-required-labels` in the plugin

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**